### ISSUE TITLE
Comment on first line in at-root directive causes error

### ISSUE THREAD

**Issue by eirikluka** [brigade/scss-lint#298]
Sass v.3.4.8
SCSS-lint v.0.30.0

``` css
@at-root {
    // comment
    .main {
    }
}
```

gives error:

``` error
SCSSLint::Linter::Indentation raised unexpected error linting file main.scss: 'undefined method `start_pos' for nil:NilClass' C:/lib/Ruby200-x64/lib/ruby/gems/2.0.0/gems/scss-lint-0.30.0/lib/scss_lint/linter/indentation.rb:130:in `at_root_contains_inline_selector?'
 C:/lib/Ruby200-x64/lib/ruby/gems/2.0.0/gems/scss-lint-0.30.0/lib/scss_lint/linter/indentation.rb:87:in `visit_atroot'
 C:/lib/Ruby200-x64/lib/ruby/gems/2.0.0/gems/sass-3.4.8/lib/sass/tree/visitors/base.rb:36:in `visit'
```
